mappings/net/minecraft/client/options/Options.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_1320 net/minecraft/client/options/GameOptions
+CLASS net/minecraft/class_1320 net/minecraft/client/options/Options
```

Comment:
Is `GameOptions` in upstream, why did u make this change?

Reply:
is Options in 1.8.9

Reply:
oh wait nvm
is Options in textile

Reply:
ll fix soon

Reply:
Then 1.8.9 is wrong

Reply:
thanks for pointing that out.

Reply:
![image](https://user-images.githubusercontent.com/25501603/105467824-ef78cb00-5ce1-11eb-9c7f-76feee955e38.png)
wait no its not

Reply:
Boogie is wrong then